lib/setup/repo.rb

```diff
@@ -0,0 +1,56 @@
+require_relative "./base"
+require_relative "../doctor/dnsmasq"
+
+class Setup::Repo < Setup::Base
+  def initialize(*args)
+    super(*args)
+    @path = File.join(ENV.fetch("HOME"), "govuk", "govuk-docker")
```

Comment:
We've got environment variable inconsistency between `govuk-docker` and the Makefile: https://github.com/alphagov/govuk-docker/blob/f56b7cc51f0ee9223b8ae2d6f158ece2774005de/Makefile#L1-L3

Can we have the same environment variables, with the same defaults?  The advantage of not hard-coding `File.join(ENV.fetch("HOME"), "govuk", "govuk-docker")` is that you can do things in a different directory.

Reply:
Yep, good call... I've added that in now.